crates/utils/sync/src/racy_lock.rs

```diff
@@ -184,4 +284,16 @@ mod tests {
         fn assert_traits<T: Send + Sync>() {}
         assert_traits::<RacyLock<Vec<i32>>>();
     }
+
+    #[test]
+    fn is_send() {
```

Comment:
Good addition of the explicit trait assertion tests! These will help catch regressions if someone accidentally changes the bounds in the future.